go/cmd/vtcombo/main.go

```diff
@@ -61,6 +61,12 @@ var (
 	mysqlPort          = flags.Int("mysql_port", 3306, "mysql port")
 	externalTopoServer = flags.Bool("external_topo_server", false, "Should vtcombo use an external topology server instead of starting its own in-memory topology server. "+
 		"If true, vtcombo will use the flags defined in topo/server.go to open topo server")
+	vschemaPersistenceDir = flags.String("vschema_persistence_dir", "", "If set, per-keyspace vschema will be persisted in this directory "+
```

Comment:
We are using `-` instead of `_` as a separator in all new flags. Can you please change this?
I know it's inconsistent, but we plan to convert all flags at some point to dashes, and making new flags use that from the beginning will reduce the amount of work needed for the conversion.